pkgs/by-name/st/styluslabs-write/package.nix

```
@@ -0,0 +1,116 @@
+{
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  stdenv,
+  pkg-config,
+  SDL2,
+  xorg,
+  libGL,
+  roboto,
+  imagemagick,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "styluslabs-write";
+  version = "2024-09-26";
+
+  src = fetchFromGitHub {
+    owner = "styluslabs";
+    repo = "Write";
+    rev = "7110e5db87c154d28efd9507be4ab768af7403e0";
+    hash = "sha256-EKYKeZg/AB10Qr0MjPGFxpCen06pNxrbTTYkh6gGnJc=";
+    fetchSubmodules = true;
+    leaveDotGit = true;
+    # Delete .git folder for better reproducibility
+    # TODO: fix GITCOUNT is always 1 but is not used in Linux Build anyway
+    postFetch = ''
+      cd $out
+      git rev-parse --short HEAD > $out/GITREV
+      git rev-list --count HEAD > $out/GITCOUNT
+      find "$out" -name .git -print0 | xargs -0 rm -rf
```

Nit: Why not simply

```suggestion
      rm -rf $out/.git
```

?

---

Copy pasted from some other file. Changed now.